src/Chr.Avro/Abstract/SchemaBuilder.cs

```diff
@@ -789,7 +813,30 @@ public override Schema BuildSchema(TypeResolution resolution, ConcurrentDictiona
                 throw new UnsupportedTypeException(resolution.Type);
             }
 
-            return cache.GetOrAdd(timestamp.Type, new StringSchema());
+            Schema schema = null;
+
+            switch (_temporalBehavior)
```

Comment:
handle the default case here (probably throw `ArgumentException`)

Reply:
https://github.com/promontis/dotnet-avro/commit/a1245d30b2ff102d154573cb1538bdcc0526a033